bedrock/firefox/templates/firefox/android/index.html

```diff
@@ -49,7 +49,7 @@ <h2 class="heading">
       </p>
 
       <div class="dl-button-wrapper">
-        <a class="dl-button" rel="external" href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}">
+        <a class="dl-button" rel="external" href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}" data-button-type="download" data-interaction="download click" data-download-version="Firefox for Android">
```

Comment:
Gareth said he can track the sticky nav CTA by ID in GTM, so no additional `data-*` attributes needed.